intro/setup/index.html

```diff
@@ -0,0 +1,349 @@
+---
+layout: default
+title: Basic window setup
+---
```

Comment:
To go with the footer, there are specific fields in this front matter that the footer is expecting (I left a note about the footer at the bottom). Specifically, the `pg` field is expecting a url to related pygame docs (if left blank, it just directs to pygame.org/docs). It's also expecting a `links` field which expecting a 2d array, where each element of the outer array is of the format `[url, title]` for referencing external resources (if left blank, will not give any external resources). In the classe page, my front matter looks like
```
---
layout: default
title: Classes in Python and Pygame
pg:
links: [[http://pythonic.zoomquiet.top/data/20070308094655/index.html, Python Types and Objects]]
---
```